woodchipper/context.py

```diff
@@ -50,4 +50,27 @@ def reset(self, token: contextvars.Token):
         self._var.reset(token)
 
 
+class LoggingContextManager:
```

Comment:
Let's rename this to LoggingContext and fix the comment below to match?

Reply:
Done with [`3f7fb58` (#4)](https://github.com/Quidge/woodchipper/pull/4/commits/3f7fb58b6f4abb23ca174d591fc7b527be502133)